### PR TITLE
LPS-169461 - NPE is thrown when exporting display page template with Content Display fragment

### PR DESCRIPTION
**Pull request by ruben-pulido**
//cc @ealonso @jkappler @lfbesada

### CODE REVIEW

**Comment by liferay-echo** [comment]
Thank you for submitting this PR. 👏

Make sure it follows the [Echo Pull Request Template](https://gist.github.com/liferay-echo/5a4455285ff1800f9da146816d85e67d#file-pull-request-template-md) if you haven't already done so. 

Learn more about it [here](https://liferay.atlassian.net/wiki/spaces/ENGECHO/pages/2027455014/Echo+Pull+Request+Etiquette).

**Comment by ruben-pulido** [comment]
ci:test:sf

**Comment by ruben-pulido** [comment]
ci:test:relevant

**Comment by lfbesada** [comment]
ci:reevaluate:1387401_4979

**Comment by ruben-pulido** [comment]
ci:test:relevant

File: modules/apps/layout/layout-impl/src/main/java/com/liferay/layout/internal/importer/structure/util/FragmentLayoutStructureItemImporter.java [hunk 1]
```diff
@@ -258,11 +258,15 @@ private FragmentEntryLink _addFragmentEntryLink(
...
+				groupId = company.getGroupId();
+			}
 		}
 
-		FragmentEntry fragmentEntry = _getFragmentEntry(
-			layout.getCompanyId(), groupId, fragmentKey);
+		FragmentEntry fragmentEntry = _getFragmentEntry(groupId, fragmentKey);
 
 		FragmentRenderer fragmentRenderer =
 			_fragmentRendererRegistry.getFragmentRenderer(fragmentKey);
```
Comment (lfbesada) [inline modules/apps/layout/layout-impl/src/main/java/com/liferay/layout/internal/importer/structure/util/FragmentLayoutStructureItemImporter.java:272]: fragmentRenderer seems to be used just for the above null checking, perhaps we can remove it. 
What do you think about it?

**Review by lfbesada — APPROVED** [review]

**Comment by ruben-pulido** [comment]
https://github.com/liferay-echo/liferay-portal/pull/10431

**Pull request closed without merging**